Tests/Microsoft.AppCenter.Test.Windows/Channel/ChannelTest.cs

```diff
@@ -113,7 +112,7 @@ public void EnableChannel()
         /// Verify that enqueuing a log passes the same log reference to enqueue event argument
         /// </summary>
         [TestMethod]
-        public void EnqueuedLogsAreSame()
+        public async Task EnqueuedLogsAreSame()
```

Comment:
`async` tests are supported, no need `RunNotAsync`. Changed it only in files which I somehow changed even without it.